server/src/main/resources/application.yaml

```diff
@@ -0,0 +1,4 @@
+blocknode:
+  storage:
+    root:
+      path: /opt/blocknode
```

Comment:
Do we want to check this in?

Reply:
Currently, application.yaml does not configure the server.  We're only able to run the server with `./gradlew run` and there's a message suggesting it's not reading the file.  While testing locally we would not want to use this path on mac.  There are instructions to override with an environment var.  This configuration was a best guess at where the root path might be.  Is there a better root path?

Reply:
I removed this yaml property for now.  We will need to revisit this once we're running the server outside of gradle.